src/auth/entities/refresh-token.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { sign } from 'jsonwebtoken'
+import { BaseEntity, Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity('refresh_tokens')
+export class RefreshToken extends BaseEntity {
+    constructor(init?: Partial<RefreshToken>) {
+        super();
+        Object.assign(this, init);
+    }
```

Comment:
I don't think extending BaseEntity is necessary since you already have @Entity decorator.
Also, you don't need to use constructors in nodejs.